src/lib/components/forms/files-input.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  let files = null;
+
+  $: () => {
+    if (files) {
+      console.log(files);
+      for (const file of files) {
+        console.log(`${file.name}: ${file.size} bytes`);
+      }
+    }
+  };
+</script>
+
+<div class="flex items-center space-x-4">
+  <label
+    for="file-upload"
+    class="muted-red cursor-pointer rounded-lg px-4 py-2 text-white hover:bg-red-600"
```

Comment:
The button differs a bit currently from the mockups:
![image](https://github.com/user-attachments/assets/e5daeceb-f936-4f05-a6b1-a7cf736d595f)
We should add the red background color to the button and maybe make that color a bit darker on hover?